core/src/main/java/io/questdb/cairo/mig/EngineMigration.java

```diff
@@ -119,7 +145,7 @@ public static void migrateEngineTo(CairoEngine engine, int latestVersion, boolea
         return MIGRATIONS.get(version);
     }
 
-    private static boolean upgradeTables(MigrationContext context, int latestVersion) {
+    private static boolean upgradeTables(MigrationContext context, int latestTableVersion, int latestMigrationVersion) {
```

Comment:
This method always returns `true`, so it could be made `void`.

Reply:
ok, done in https://github.com/questdb/questdb/commit/38a1993364bdf752a51f621b382d320b6550f615